### ISSUE TITLE
Installation via Carthage is failing

### ISSUE THREAD

**Issue by artemnovichkov** [hirokimu/EMTLoadingIndicator#3]
My Cartfile:
```
github "hirokimu/EMTLoadingIndicator" ~> 4.0
```
When I run `carthage update EMTLoadingIndicator --platform watchos`, I have an error:
```
*** Skipped building EMTLoadingIndicator due to the error:
Dependency "EMTLoadingIndicator" has no shared framework schemes for any of the platforms: watchOS

If you believe this to be an error, please file an issue with the maintainers at https://github.com/hirokimu/EMTLoadingIndicator/issues/new
```

**Comment by hirokimu** [comment]
@artemnovichkov I fixed this issue on 4.0.1. Thank you.

**Comment by zalewskise** [comment]
> @artemnovichkov I fixed this issue on 4.0.1. Thank you.

Please update readme file.
